core/domain/domain.fr.yml

```diff
@@ -835,3 +835,30 @@ responses:
 
   utter_daily_ci__early_opt_out__cancel_ci_recommendation:
     - text: Si vous remarquez des changements, ou si vous avez des symptômes, n'hésitez pas à communiquer avec moi pour une évaluation
+
+  ## Daily check-in invalid ID
+
+  utter_daily_ci__invalid_id__invalid_link:
+    - text: Je suis désolée mais ce lien ne semble pas valide
+
+  utter_daily_ci__invalid_id__try_again:
+    - text: Veuillez essayer de nouveau, et si vous copiez le lien dans votre fureteur, assurez-vous de copier exactement la même chose
+
+  utter_ask_daily_checkin__invalid_id__want_assessment:
+    - text: Si ça ne fonctionne pas, voudriez-vous évaluer vos symptômes?
+      buttons:
+        - title: Oui
+          payload: "/affirm"
+        - title: Non
+          payload: "/deny"
+
+  utter_acknowledge_ask_anything_else:
+    - text: "D'accord, puis-je faire autre chose pour vous aider?"
+      buttons:
+        - title: J'ai des questions
+          payload: "/ask_question"
+        - title: Non j'ai terminé
+          payload: "/done"
```

Comment:
Is this a generic message or not? It should either be prefixed or placed in the generic messages section.

Reply:
Generic. Didn't realize we had a "generic" section ;) We'll have to see if there is something we can to with all these "what next" to help, or at least not hinder, the ML one day.

Reply:
Finally, I prefixed it... What is offered depends on where we are so it is not so generic.